upgrade_pythoncapi.py

```diff
@@ -583,10 +591,15 @@ def main(self):
 
         if self.pythoncapi_compat_added and not self.args.quiet:
             self.log()
-            self.log(f"{INCLUDE_PYTHONCAPI_COMPAT} added: you may have "
-                     f"to copy {PYTHONCAPI_COMPAT_H } to your project")
-            self.log("It can be copied from:")
-            self.log(PYTHONCAPI_COMPAT_URL)
+            if self.args.download_latest_header:
+                for path in self.args.paths:
```

Comment:
It can be a list of directories and files. It may download the file multiple times and write it multiple times in the same directory, no?

Reply:
Also, it copies the files in directory in which no files were modified. It's sub-optimal.